src/Microsoft.Azure.SignalR.Common/ServiceConnections/ServiceConnectionBase.cs

```diff
@@ -675,24 +666,18 @@ private static class Log
             private static readonly Action<ILogger, Exception> _handshakeComplete =
                 LoggerMessage.Define(LogLevel.Debug, new EventId(22, "HandshakeComplete"), "Handshake with service completes.");
 
-            private static readonly Action<ILogger, string, Exception> _errorReceivingHandshakeResponse =
-                LoggerMessage.Define<string>(LogLevel.Error, new EventId(23, "ErrorReceivingHandshakeResponse"), "Error receiving handshake response. Id: {ServiceConnectionId}");
-
-            private static readonly Action<ILogger, string, string, Exception> _handshakeError =
-                LoggerMessage.Define<string, string>(LogLevel.Critical, new EventId(24, "HandshakeError"), "Service returned handshake error: {Error}. Id: {ServiceConnectionId}");
+            private static readonly Action<ILogger, string, string, string, Exception> _handshakeError =
+                LoggerMessage.Define<string, string, string>(LogLevel.Information, new EventId(24, "HandshakeError"), "Connection to service '{Endpoint}' returned handshake error, probably caused by network instability or service restart. Will retry after the back off period. Error detail: {Error}. Id: {ServiceConnectionId}");
```

Comment:
we define "Error" in service, should we keep the same?

Reply:
Error means the customers receive alerts and they need to do action to solve it. Since customers can do little when the connection drops and actually it has little impact on the customer, Info sounds reasonable.

Reply:
We have warning logs when all the connections to the endpoints are dropped. -- and it is when there is real impact to the customer.

Reply:
Error/warning is a way to notify customer there is/likely to be a problem no matter customer can do action? and Info should be expected behavior like xxx success/start/end?

Reply:
Yeah so the issue here is there is no real problem here so we don't want to notify customers. Maybe we should change it to "Verbose"?

Reply:
`Debug` sounds make more sense. Changed

Reply:
Offline discussed. We will keep connection related logs information, customer to aware logs warning, and application exceptions error.